test/core/extensions/io_extension_test.cpp

```diff
@@ -18,6 +19,9 @@ using kagome::common::Buffer;
 using kagome::runtime::MockMemory;
 using kagome::runtime::WasmPointer;
 using kagome::runtime::WasmSize;
+using kagome::runtime::WasmLogLevel;
+using kagome::runtime::WasmResult;
+using kagome::runtime::WasmEnum;
```

Comment:
To big list... We can simplify by `using namespace` statement.
```suggestion
using namespace kagome::runtime;
```

Reply:
No ```using namepace``` makes visible all types of the namespace. It's a bad practice, that can cause ambiguity. ```using type``` much better. If I'm wright clang doesn't allow you to do this without warning.